WeScanTests/CIRectangleDetectorTests.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CIRectangleDetectorTests.swift
+//  WeScanTests
+//
+//  Created by James Campbell on 8/8/18.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import XCTest
+@testable import WeScan
+
+class CIRectangleDetectorTests: XCTestCase {
+    
+    func testDetectorUsesValidType() {
+        XCTAssertNotNil(CIRectangleDetector.rectangleDetector)
```

Comment:
This test is not really matching its description, it only calls one method.

How do we know it's using the valid type here?

Reply:
@AvdLee this is mainly to detect that the static variable is always set to use the rectangle detector, there doesn't seem to be a way to actually inspect the type used by the CI class used for detection itself.

The only solution for that would be some sort of test that feeds an image with a rectangle and verifies that the CI is correctly configured to give us a rectangle result.

Reply:
That sounds a lot better than this test imo! If we could create a test like that, it would be great 👍